### ISSUE TITLE
xcf binary mask next steps

### ISSUE THREAD

**Issue by JacobGlennAyers** [JacobGlennAyers/spectrace#5]
Right now I have the repository set up that if someone draws frequency traces using gimp, we get out an xcf file with the image and all of the layer information. I have a script that takes those xcf files and converts them into binary masks and outputs a visualization as a proof of concept.

The natural next step is to write some scripts that accomplish these two tasks - 

1. Allows a user to get time-frequency tables based on the binary masks for each layer group.
2. Write a script that "prepares" the binary masks for machine learning. Preferably using some kind of PyTorch dataset + dataloader class of some kind?
